components/Users/FoodEntriesPerEmail.jsx

```diff
@@ -0,0 +1,27 @@
+// reusable component that calls /api/foodEntriesPerEmail api to get food entries per email
+
+
+import axios from "axios";
+import useSWR, { mutate, useSWRConfig } from "swr";
+
+const fetcherForUserInfo = (url, userEmail) =>
+  axios
+    .get(url, { params: { userEmail: userEmail } })
+    .then((response) => response.data);
+
+export const FoodEntriesPerEmail = ( props ) => {
+  
+  const {
+    data: foodEntriesPerEmail,
+    error,
+    isLoading,
+    mutate: mutateFoodPerEmail,
+  } = useSWR(["/api/foodEntriesPerEmail",props.sessionEmail], fetcherForUserInfo);
```

Comment:
Move `/api/foodEntriesPerEmail` url to a separate file (for centralized) like - `lib/urls/food.js`